docs/user.md

```diff
@@ -28,15 +28,21 @@ Table Of Contents:
 
 ## Process <a name="process"></a>
 
-More detail on each step can be found in the sections further below.
+Overview of the process to apply the results to the VCSP Program:
```

Comment:
We need to wait for the Program manager to check if we need to mention the program here.
https://coreos.slack.com/archives/C02DSMAP4LA/p1670882046186029